README.md

```diff
@@ -113,6 +115,56 @@ Finally, properties are what give the selected elements of a rule declaration th
   - Uses of z-index
   - Compatibility or browser-specific hacks
 
+* Prefer line comments (`//` in Sass-land) to block comments.
+* Prefer comments on their own line. Avoid end-of-line comments.
+* Write detailed comments for code that isn't self-documenting:
+  - Uses of z-index
+  - Compatibility or browser-specific hacks
+
+### Colors
+
+When implementing feature styles, you should only be using color variables provided by theme.css
+
+When adding a color variable to theme.css, only use colors available to you by
+the collor pallete. Those should be listed to you in a separate file called
+`color_pallete.css`.
+
+Using HEX color units is preffered over RGB, RGBA, named, HSL, or HSLA values.
+
+**Bad**
+
+```css
+rgb(50, 50, 50);
+rgba(50, 50, 50, 0.2);
```

Comment:
When its defining the color at the same time yes. Because now we have this random color that isn't a variable. I would only use this when you absolutely can't get away with opacity property.